### ISSUE TITLE
when you set feed_type=FEED_HTTP in FirmAFL_config, the fuzzer can't work

### ISSUE THREAD

**Issue by minifish120** [zyw-200/FirmAFL#30]
when you set feed_type=FEED_HTTP in FirmAFL_config, the fuzzer can't work. 
i followed the code, and found that in the user_mode/linux-user/main.c, the main() function invoked  prepare_feed_input(cpu), but in the prepare_feed_input, it didn't deal with  the condition when feed_type=FEED_HTTP. May be it's a design bug?

**Comment by zyw-200** [comment]
When feeding type is FEED_HTTP, the input feeding logic is implemented in function feed_input()
The function prepared_feed_input only serves the FEED_ENV and FEED_CMD feeding type.

**Comment by minifish120** [comment]
yes, i see you update your code, but it still doesn't work for FEED_HTTP in my experiment, i don't know what's the problem for me.
take the 9925 as example, when i first run run.sh, secondly run "python test.py", and finally run "./user.sh", it reports:
cp:./afl-qemu-trace
[*] Attempting dry run with 'id:000000,orig:seed'...
[*] Spinning up the fork server...
syn_shmem_id:7
hva_start:7fa19acd2000
mem file mem_file
main is:httpd
open write pipe:4
open read pipe:5,9096
snapshot memory:3d0027,80200000,80208000
snapshot phys addr:7, 7f7965af8000, ffffffffffffffff
......
and the system qemu exited abnormally
......
so what's the problem?

i changed to the 10853, the phenomenon is the same, too

**Comment by minifish120** [comment]
and i read your code, found it didn't print the line "snapshot phys addr end.\n" in user_mode/linux-user/main.c:

//sync_shmem_start = shmat(syn_shmem_id, NULL,  1); 
      printf("snapshot phys addr:%d, %lx, %lx\n",syn_shmem_id, guest_base, sync_shmem_start);
      memset(sync_shmem_start, 0, 8192); //131071
      //phys_addr_stored = (int *)shmem_start; // make a big mistake here, cause two shared memory overlapping
      phys_addr_stored_bitmap = (char *)sync_shmem_start;
      printf("snapshot phys addr end.\n");
